server/sitemap.ts

```diff
@@ -25,9 +26,23 @@ const generateSitemapPage = (
   root: string,
   { loc, lastmod = defaultLastmod, changefreq = "daily", priority }: SitemapPage
 ) => {
+  const firstLvlNav = findFirstLvlNav(loc);
+  let updateLoc = "";
+
+  switch (firstLvlNav) {
+    case "cloud":
+      updateLoc = `${updateLoc}?scope=cloud`;
+      break;
+    case "enterprise":
+      updateLoc = `${updateLoc}?scope=enterprise`;
+      break;
+    default:
+      updateLoc = loc;
+  }
```

Comment:
I think you're missing out the original path when the switch/case operates, you're only adding an empty string and the scope.

<img width="504" alt="Screen Shot 2022-07-11 at 9 13 30 AM" src="https://user-images.githubusercontent.com/73362670/178320387-4fecb3a4-7a25-4cb3-8f4a-d6d32428b231.png">

<img width="494" alt="Screen Shot 2022-07-11 at 9 15 10 AM" src="https://user-images.githubusercontent.com/73362670/178320654-bbca305a-facd-4c42-aec4-9562a84c2bce.png">